src/scicookie/{{cookiecutter.project_slug}}/build-system/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "{{ cookiecutter.project_slug }}"
+version = "0.1.0"
+authors = ["{{ cookiecutter.author_full_name }} <{{ cookiecutter.author_email }}>"]
+
+[lib]
+name = "_core"
+# "cdylib" is necessary to produce a shared library for Python to import from.
+crate-type = ["cdylib"]
+
+[dependencies]
+rand = "0.8.4"
+
+[dependencies.pyo3]
+version = "0.19.1"
+# "extension-module" tells pyo3 we want to build an extension module (skips linking against libpython.so)
+# "abi3-py38" tells pyo3 (and maturin) to build using the stable ABI with minimum Python version 3.8
+features = ["extension-module", "abi3-py38"]
```

Comment:
here it says it is using python3.10: https://github.com/osl-incubator/scicookie/actions/runs/5684760045/job/15408166724?pr=152#step:5:4275
so probably it should be `abi3-py310` ... but it would be nice to get there from somewhere in order to be dynamic

Reply:
ohhh sorry .. I missed `ABI with minimum Python version 3.8` ... so probably my comment before doesn't make any sense XD sorry